ac/ac-quiz/src/LatexWidget.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import React from 'react';
+import Latex from 'react-latex';
+
+export default (props: Object) => {
+  const { options, value, required, disabled, readonly, onChange } = props;
+  // Generating a unique field name to identify this set of radio buttons
+  const name = Math.random().toString();
```

Comment:
why don't you use uuid()?:)

Reply:
That the code I copied from the RJSF RadioWidget. I fixed a few eslint errors on it, but didn't look at it much more. I guess the safest is to leave it as it is.